oauth_provider/oauth2/validator.py

```diff
@@ -185,7 +185,8 @@ def save_bearer_token(self, token, request, *args, **kwargs):
             'expires_at': fields.Datetime.to_string(
                 datetime.now() + timedelta(seconds=token['expires_in'])),
         })
-        return request.client.redirect_uri_ids[0].name
+        redirect_uris = request.client.redirect_uri_ids
+        return redirect_uris and redirect_uris[0].name or ''
```

Comment:
`return request.client.redirect_uri_ids[:1].name` seems easier to read to me, what's your opinion about that ?
This syntax is sometimes used in the standard code, and I find this trick really useful to avoid many if/else/and/or.

Reply:
Totally agree!